config/galaxy.ini

```diff
@@ -1157,4 +1434,3 @@ admin_users = phenomenal@phenomenal-h2020.eu
 #exchange = galaxy_exchange
 #routing_key = bar_code_scanner
 #rabbitmqctl_path = /path/to/rabbitmqctl
```

Comment:
We should explore using rabbitmq for message passing when we move to the multiple web servers and multiple handlers setting.